TWTValidation/JSON Validator/Parser/TWTJSONSchemaParser.m

```diff
@@ -672,7 +726,7 @@ - (void)failIfObject:(id)object isNotMemberOfSet:(NSSet *)validValues
 
 - (void)failWithErrorCode:(NSUInteger)code object:(id)object format:(NSString *)format, ...
 {
-    NSParameterAssert(object);
+    object ? object : [NSNull null];
```

Comment:
No, I mean you’re not actually doing anything with the result. The line doesn’t reassign object or anything.